src/parquet/arrow/writer.h

```diff
@@ -132,6 +132,14 @@ class PARQUET_EXPORT FileWriter {
       const std::shared_ptr<ArrowWriterProperties>& arrow_properties,
       std::unique_ptr<FileWriter>* writer);
 
+  static ::arrow::Status WriteMetaData(
+      const std::unique_ptr<FileMetaData>& fileMetaData,
+      const std::shared_ptr<OutputStream>& sink);
+
+  static ::arrow::Status WriteMetaData(
+      const std::unique_ptr<FileMetaData>& fileMetaData,
+      const std::shared_ptr<::arrow::io::OutputStream>& sink);
```

Comment:
Parameter names are non-conforming to our style guide. Also, we should pass sink as `arrow::io::OutputStream* sink` instead of a shared_ptr. I opened https://issues.apache.org/jira/browse/PARQUET-1360